### DISCUSSION TITLE
Release Cadence

### DISCUSSION THREAD

**Discussion by thechucklingatom** [acacode/swagger-typescript-api#982] [General]
I see there was quite a few releases in August, but nothing since the end of August. Is there a projected timeline for when we can expect a new release?

**Comment by smorimoto** [comment]
We are in the process of performing a major refactoring and the next release may take a little more time. I think we can make alpha releases if you want to try.

**Comment by thechucklingatom** [comment]
No worries, I understand. Do you have an expected release range for the refactor? I have some colleagues that are waiting for #930 which I see is merged into master. I wasn't sure if there was a plan to make any minor versions releases. I would be happy to test an alpha release with our use case, assuming that the lift for upgrading is not too significant.

**Reply by smorimoto** [reply]
Oh, I get it. I will try to make a release with just that backported!

**Reply by smorimoto** [reply]
@thechucklingatom https://github.com/acacode/swagger-typescript-api/releases/tag/13.0.23

**Reply by thechucklingatom** [reply]
Thank you so much!
